main.go

```diff
@@ -51,12 +60,48 @@ func main() {
 	}
 	err := c.Read(configFile)
 	if err != nil {
-		log.Error(err)
-		return
+		log.Fatal(err)
 	}
 
-	signatory := signatory.NewSignatory(createVaults(c), &c.Tezos, metrics.IncNewSigningOp)
+	vaults, healths := createVaults(c)
+	signatory := signatory.NewSignatory(vaults, &c.Tezos, metrics.IncNewSigningOp)
+
+	srv := server.NewServer(signatory, &c.Server)
+	utilityServer := server.NewUtilityServer(&c.Server, healths)
+
+	go func() {
+		err := utilityServer.Serve()
+		if err != nil {
+			errChan <- err
+		}
+	}()
+
+	log.Infof("Utility Server listening on port: %d", c.Server.Port+1)
```

Comment:
Let's make the health check port independently configurable, that will be less magical.

We can choose a default port, and register it here; https://github.com/prometheus/prometheus/wiki/Default-port-allocations

Reply:
I allocated port `9583` for signatory over at: https://github.com/prometheus/prometheus/wiki/Default-port-allocations